autoload/jedi.vim

```diff
@@ -233,6 +242,14 @@ call jedi#init_python()  " Might throw an error.
 " ------------------------------------------------------------------------
 " functions that call python code
 " ------------------------------------------------------------------------
+function! jedi#use_environment(...) abort
+  PythonJedi jedi_vim.set_environment(*vim.eval('a:000'))
+endfunction
+
+function! jedi#use_environment_for_buffer(env) abort
```

Comment:
Same as above.
There should also be setters (and getters) for tab- and window-local settings.